great_tables/_export.py

```diff
@@ -166,6 +168,30 @@ def as_raw_html(
     return html_table
 
 
+def as_latex(self: GT) -> str:
+    """
+    Output a GT object as LaTeX
+
+    Get the LaTeX content from a GT object as a string.
+
+    Parameters
+    ----------
+    gt
+        A GT object.
+
+    Returns
+    -------
+    str
+        An LaTeX fragment containing a table.
+
+    """
+    built_table = self._build_data(context="latex")
```

Comment:
I think the comment on `great_tables/_text.py` refers to an older version. Could you check the latest version of `great_tables/_text.py`? I'm using `.to_*()` methods there.

Reply:
Oh shoot -- I see them now, sorry for the mix up 😓

Reply:
It's okay! Just moved `migrate_unformatted_to_output()` to `utils.py` for now. And it's a function now. There are some type annotation issues that I flagged in a TODO comment (circular dependency issue when importing the parameter types). This could perhaps be resolved by placing in a different file (will experiment later).